src/language/server.ts

```diff
@@ -108,7 +109,7 @@ connection.onInitialize((_params) : InitializeResult => {
   shouldSendDiagnosticRelatedInformation = _params.capabilities && _params.capabilities.textDocument && _params.capabilities.textDocument.publishDiagnostics && _params.capabilities.textDocument.publishDiagnostics.relatedInformation;
   return {
     capabilities: {
-      textDocumentSync: documents.syncKind,
+      textDocumentSync: TextDocumentSyncKind.Full,
```

Comment:
I am not sure whether `SyncKind` should be `Full` or `Incremental`.

Reply:
I'm not sure either.  In my own test upgrades, I was using Incremental - but I didn't get very far in the testing, and upon 2nd thoughts - Full might be the better option here.